### PR TITLE
[Correction] Adding R packages raised in the issue #2831

### PR DESCRIPTION
**Pull request by balajialg**
@yuvipanda, Sorry for the confusing messaging with the last commit. For some reason, the changes for installing R packages did not get added as part of my previous commit. I realized it now and correcting it with this PR!

### CODE REVIEW

File: deployments/publichealth/image/r-packages/ph-252.r [hunk 1]
```diff
@@ -16,6 +16,12 @@ class_libs = c(
   "geepack", "1.3-2",
   "multcomp", "1.4-17",
   "epitools", "0.5-10.1"
+  "rms", "6.2-0",
+  "lmtest","0.9-38",
+  "car","3.0-11",
+  "SurvRegCensCov","1.4",
+  "ProfileLikelihood","1.1"
```
Comment (yuvipanda) [inline deployments/publichealth/image/r-packages/ph-252.r]:
```suggestion
  "ProfileLikelihood","1.1",
```

File: deployments/publichealth/image/r-packages/ph-252.r [hunk 2]
```diff
@@ -16,6 +16,12 @@ class_libs = c(
   "geepack", "1.3-2",
   "multcomp", "1.4-17",
   "epitools", "0.5-10.1"
```
Comment (yuvipanda) [inline deployments/publichealth/image/r-packages/ph-252.r]:
```suggestion
  "epitools", "0.5-10.1",
```

**Review by yuvipanda — COMMENTED** [review]
Missing commas are causing the installation to fail

**Comment by yuvipanda** [comment]
@balajialg Based on the test failure, it looks like one of the versions for a package isn't available?

**Comment by balajialg** [comment]
@yuvipanda Moved to the previous version of the car package and retriggered the build. If this doesn't work, Can you recommend any other next steps?

**Comment by balajialg** [comment]
Thankfully, Seems to have worked!

**Pull request merged**